### DISCUSSION TITLE
SFTP: Auth fail

### DISCUSSION THREAD

**Discussion by rjkroll** [nextgenhealthcare/connect#4920] [Q&A]
Hi,

Running into a wierd one today.    I have a Channel that is just a basic SFTP file reader where it goes out and pulls down files.    No keys involved, just using a username and password to connect.   It was working for for the last two years, now though, I get random 'Auth fail' errors when trying to Poll, but then it will randomly work.  

The real problem, though, is that I am also getting these "jcraft.jsch.JSchException: Auth fail" errors when trying to perform the 'After Processing Action', which I have marked as DELETE.    So, the channel will pull down some files, not be able to Delete the files from the SFTP server, then it just gets stuck in a 'Reading' state until I kill it.

I have the following settings:
File Reader -> SFTP
Authentication :  Password
Host Name Checking :  Ask
..and nothing for known hosts file or Configuration Options.

The outside company indicates "no changes on their end" and I am able to connect via WinSCP or by using a .batch file script, so I'm not too sure if this is a bug or something else is happening.

Mirth Connect v3.4.2
Java 1.8

**Comment by jonbartels** [comment]
Are you able to exactly replicate the order of operations from WinSCP or a script?

Can you ask the outside company if their end is logging authentication failures? Even if they have no changes, seeing if this is a genuine auth failure from the server side would help. It could be some other error which manifests as auth fail such as authentication succeeding but losing a connection before a response is sent.

Taking a packet capture might help, you obviously won't be able to see the traffic but you may see dropped connections or timeout issues.
